src/Calculator/Common/KeyboardShortcutManager.cpp

```diff
@@ -642,6 +642,13 @@ void KeyboardShortcutManager::OnAcceleratorKeyActivated(CoreDispatcher ^, Accele
             return;
         }
 
+        // Ctrl is pressed in addition to alt, this means Alt Gr is intended.  do not navigate.
+        if ((static_cast<short>(Window::Current->CoreWindow->GetKeyState(VirtualKey::Control)) & static_cast<short>(CoreVirtualKeyStates::Down))
```

Comment:
Is there a way for us to detect the keyboard type and add that to the if condition?

Reply:
Even if qwerty keyboards don't have a `Altr Gr` button, Ctrl and Alt will also do some special actions. A good example is Ctrl+Alt+C, it won't   copy but type `©`. We should not try to check keyboard layouts.

Reply:
That makes sense to me. I just wanted to make sure we weren't blocking user intent in other cases.